### DISCUSSION TITLE
Add FFXIV launcher?

### DISCUSSION THREAD

**Discussion by PyroKitty7** [moraroy/NonSteamLaunchers-On-Steam-Deck#167] [Ideas]
Those of us that don't own Final Fantasy XIV through Steam have to go through crazy hoops right now in order to load the non-Steam launcher so we can play on the Deck. I wonder if there's a way to add that launcher to this? That would be amazing if there were!

**Comment by moraroy** [comment]
@PyroKitty7 it's possible!! I just haven't looked into it yet haha
